### ISSUE TITLE
In a ganttchart, trying to hide the taskaxis gridlines hides the entire chart

### ISSUE THREAD

**Issue by yusufnazir** [syampillai/SOCharts#19]
`ganttChart.getTaskAxis().getGridLines(true).hide();`
The moment i call this code, the chart isn't visible anymore. I went through the code, but don't see why it is hiding the chart.

**Comment by yusufnazir** [comment]
Ok i debugged the splitLine issue and saw the json isn't generated correctly:
```java
  "splitLine": {
        "id": 26,
        "show": false,
      },
``` 
There is a comma before the closing bracket.
Traced this back to:
```java
public static class GridLines extends Line {

        private int interval = Integer.MIN_VALUE;

        /**
         * Constructor.
         */
        public GridLines() {
        }

        @Override
        public void encodeJSON(StringBuilder sb) {
            super.encodeJSON(sb);
            ComponentPart.addComma(sb);
            if(interval >= -1) {
                sb.append("\"interval\":");
                if(interval == -1) {
                    sb.append("\"auto\"");
                } else {
                    sb.append(interval);
                }
            }
        }
``` 

If i move the "ComponentPart.addComma(sb);" after the line "  if(interval >= -1) {" then is does work.

**Comment by syampillai** [comment]
Thanks for reporting. I have corrected this in version 3.2.3
